src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -88,51 +90,51 @@ public Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext,
 
         private async Task<bool> IsEnabledWithVariantsAsync<TContext>(string feature, TContext appContext, bool useAppContext, CancellationToken cancellationToken)
         {
-            bool isFeatureEnabled = await IsEnabledAsync(feature, appContext, useAppContext, cancellationToken).ConfigureAwait(false);
+            bool isFeatureEnabled = false;
 
-            FeatureDefinition featureDefinition = await _featureDefinitionProvider.GetFeatureDefinitionAsync(feature).ConfigureAwait(false);
+            FeatureDefinition featureDefinition = await GetFeatureDefinition(feature).ConfigureAwait(false);
 
-            if (featureDefinition == null || featureDefinition.Status == FeatureStatus.Disabled)
-            {
-                isFeatureEnabled = false;
-            }
-            else if ((featureDefinition.Variants?.Any() ?? false) && featureDefinition.Allocation != null)
+            VariantDefinition variantDefinition = null;
+            TargetingContext targetingContext = null;
```

Comment:
Should have a newline between the variable declarations for spacing consistency.